_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -97,18 +89,17 @@ function add_noscript_head_meta() {
 		echo '</noscript>';
 	}
 
+	/**
+	 * Custom menu order.
+	 *
+	 * @deprecated since 9.1.0
+	 * @param array $menu_order Menu order.
+	 * @return array
+	 */
 	function jetpack_menu_order( $menu_order ) {
-		$jp_menu_order = array();
-
-		foreach ( $menu_order as $index => $item ) {
-			if ( $item != 'jetpack' )
-				$jp_menu_order[] = $item;
-
-			if ( $index == 0 )
-				$jp_menu_order[] = 'jetpack';
-		}
+		_deprecated_function( __METHOD__, 'jetpack-9.1' );
```

Comment:
```suggestion
		_deprecated_function( __METHOD__, 'jetpack-9.2' );
```

We're past the branch cut for 9.1. Let's update this to 9.2.